src/base/Cellar.sol

```diff
@@ -1079,23 +1158,67 @@ contract Cellar is ERC4626, Ownable, Multicall {
     /**
      * @notice Transfer accrued fees to the Sommelier chain to distribute.
      * @dev Fees are accrued as shares and redeemed upon transfer.
+     * @dev assumes cellar's accounting asset is able to be transferred and sent to Cosmos
      */
     function sendFees() public onlyOwner {
+        require(strategistPayoutAddress != address(0), "Strategist Payout not set");
+        uint256 _totalAssets = totalAssets();
+
+        // Since this action mints shares, calculate outstanding performance fees due.
+        _takePerformanceFees(_totalAssets);
+
+        uint256 currentBalance = balanceOf[address(this)];
+
+        console.log("Performance Fee Assets", _convertToAssets(currentBalance, _totalAssets)); // This is right
+
+        uint256 strategistFeeSharesDue = currentBalance.mulDivDown(strategistPerformanceCut, 1e18);
```

Comment:
Use `mulWadDown` instead of `mulDivDown` for all the instances where you are dividing by `1e18`.

I counted three places (including this one) where this should be changed.